struct.rkt

```diff
@@ -62,3 +90,53 @@
 (provide *Range
          make-Range
          Range->hash)
+
+(struct FormattingOptions
+  (tabSize
+   insertSpaces
+   trimTrailingWhitespace
+   insertFinalNewline
+   trimFinalNewlines
+   key))
```

Comment:
Not sure, but do we always use this style for structure fields? Or JSON requires that?

Reply:
It's intended to be same as the names in LSP specification.

Maybe we can have a macro to automatically converts these names to racket name style.
Then we only use the LSP style names in `jsexpr->FormattingOptions`, and racket style names everywhere else.

Reply:
It would be great if that is possible

Reply:
What about `FormattingOptions` and other top level structs? Their name are also inconsistent with Racket style.

Reply:
Same, but I think maybe let's have another PR for these changes.

Reply:
ok. I'm going to rename these fields.